console/console/build_command.go

```diff
@@ -38,70 +37,99 @@ func (receiver *BuildCommand) Extend() command.Extend {
 		Category: "build",
 		Flags: []command.Flag{
 			&command.StringFlag{
-				Name:    "system",
+				Name:    "os",
+				Aliases: []string{"o"},
+				Value:   "",
+				Usage:   "Target os",
+			},
+			&command.BoolFlag{
+				Name:    "static",
 				Aliases: []string{"s"},
+				Value:   false,
+				Usage:   "Static compilation",
+			},
+			&command.StringFlag{
+				Name:    "name",
+				Aliases: []string{"n"},
 				Value:   "",
-				Usage:   "target system os",
+				Usage:   "Output binary name",
 			},
 		},
 	}
 }
 
 // Handle Execute the console command.
 func (receiver *BuildCommand) Handle(ctx console.Context) error {
+	var err error
 	if receiver.config.GetString("app.env") == "production" {
-		color.Yellow().Println("**************************************")
-		color.Yellow().Println("*     Application In Production!     *")
-		color.Yellow().Println("**************************************")
+		ctx.Warning("**************************************")
+		ctx.Warning("*     Application In Production!     *")
+		ctx.Warning("**************************************")
 
 		answer, err := ctx.Confirm("Do you really wish to run this command?")
 		if err != nil {
-			return err
+			ctx.Error(fmt.Sprintf("Confirm error: %v", err))
+			return nil
 		}
 
 		if !answer {
-			color.Yellow().Println("Command cancelled!")
+			ctx.Warning("Command cancelled!")
 			return nil
 		}
 	}
 
-	system := ctx.Option("system")
-	validSystems := []string{"linux", "windows", "darwin"}
-	isValidOption := func(option string) bool {
-		for _, validOption := range validSystems {
-			if option == validOption {
-				return true
-			}
+	os := ctx.Option("os")
+	if os == "" {
+		os, err = ctx.Choice("Select target os", []console.Choice{
+			{Key: "Linux", Value: "linux"},
+			{Key: "Darwin", Value: "windows"},
+			{Key: "Windows", Value: "darwin"},
```

Comment:
it doesn't seem to match in darwin and windows value

Reply:
Good catch